fleet/api/grpc/endpoint.go

```diff
@@ -12,11 +12,30 @@ func retrieveAgentEndpoint(svc fleet.Service) endpoint.Endpoint {
 		if err := req.validate(); err != nil {
 			return nil, err
 		}
-		agentGroup, err := svc.ViewAgentGroupByIDInternal(ctx, req.OwnerID, req.AgentID)
+		agent, err := svc.ViewAgentByIDInternal(ctx, req.OwnerID, req.AgentID)
 		if err != nil {
 			return nil, err
 		}
 		res := agentRes{
+			id:      agent.MFThingID,
+			name:    agent.Name.String(),
+			channel: agent.MFChannelID,
```

Comment:
I think you set this up to get the channel, but we know we don't need that sense policies will not be using it, so do we need this still?